media-video/shotcut/shotcut-19.12.31.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A free, open source, cross-platform video editor"
+DESCRIPTION="Major features include support for a wide range of formats."
+HOMEPAGE="https://shotcut.org/"
+COPYRIGHT="2011-2019 by Meltytech, LLC."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/mltframework/shotcut/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d7017d605edb581cb47de4844827bd627c159c810614381f4586ef35e8c3d97a"
+SOURCE_DIR="shotcut-$portVersion"
+PATCHES="shotcut-$portVersion.patchset"
+
+ARCHITECTURES="?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	shotcut$secondaryArchSuffix = $portVersion
+ 	app:shotcut
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:ffmpeg
+	cmd:melt
+	frei0r$secondaryArchSuffix
+	ladspa_sdk$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libmlt++$secondaryArchSuffix
+	lib:libmlt$secondaryArchSuffix
+	lib:libQt5$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Designer$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Sql$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	lib:libQt5WebKitWidgets$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libwebm$secondaryArchSuffix
+	lib:libx264$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libmlt++$secondaryArchSuffix
+	devel:libmlt$secondaryArchSuffix
+	devel:libQt5$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Designer$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Sql$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libwebm$secondaryArchSuffix
+	devel:libx264$secondaryArchSuffix
+	frei0r${secondaryArchSuffix}_devel
+	ladspa_sdk${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:getconf
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	qmake PREFIX=$prefix
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	mkdir -p $appsDir $dataDir
+	mv $prefix/share/shotcut $dataDir
+	mv $prefix/share/metainfo $dataDir
+	mv $prefix/share/applications $dataDir
+	mv $prefix/share/mime/packages $dataDir
+	mv $prefix/share/icons/hicolor/64x64/apps $dataDir
+	mv $prefix/share/man/man1 $dataDir
+	rm -rf $prefix/share
```

Comment:
applications and icons shouldn't be needed and can be removed
man1 should go to $manDir